src/main/java/uk/gov/pay/adminusers/model/StripeAgreementRequest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.pay.adminusers.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.pay.adminusers.resources.ValidIpAddress;
+
+public class StripeAgreementRequest {
+    
+    @ValidIpAddress
+    private final String ipAddress;
+    
+    @JsonCreator
+    public StripeAgreementRequest(@JsonProperty("ip_address") String ipAddress) {
```

Comment:
Can we use `JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)` instead and not have to explicitly snake-case each property?